src/lib/pipeline.ml

```diff
@@ -74,6 +74,7 @@ module Parameters = struct
     with_mutect2: bool [@default false];
     with_varscan: bool [@default false];
     with_somaticsniper: bool [@default false];
+    with_optitype: bool [@default false];
```

Comment:
I like that it's optional here, but you need to implement that logic below (and through the report, as with `rna_bam` etc. Fine if it runs by default, though `[@default true]`. Not sure OTOMH how to resolve the confusion between seq2hla's results being fed through the rest of the pipeline, but optitype's being completely ignored (except for the sake of the report). 

One idea (and 100% fine with it not being addressed here, but if not, could you file an issue?), is to add a CLI option like `--hla-type-for-epitope-prediction` as an enum type one of [Optitype, Seq2Hla, Explicit (i.e. we pass the HLA types to the pipeline directly)] with some sort of fail-fast runtime error check that those options exist (i.e. if we specify optitype, that we're running optitype, likewise for Seq2Hla, and similarly that if we specify "explicit", that we're passing HLA types on the CLI). There may be a cleaner way to handle this, though; adding more and more CLI flags begins to get scary, quickly…